CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Added `useInfiniteScroll` props for infinite scroll support in `QasListView`.
+- Added `infiniteScrollProps` props passed to `QInfiniteScroll` component.
```

Comment:
`- Added `infiniteScrollProps` prop for sending to `QInfiniteScroll` component.` melhor não?

Reply:
feito